packages/chain-adapters/src/utxo/utxoSelect/index.test.ts

```diff
@@ -1,29 +1,49 @@
 import { utxoSelect } from './'
 
+const baseUtxos = [
+  {
+    txid: 'ef935d850e7d596f98c6e24d5f25faa770f6e6d8e5eab94dea3e2154c3643986',
+    vout: 0,
+    value: '1598',
+    height: 705718,
+    confirmations: 2,
+    address: 'bc1qpszctuml70ulzf7f0zy5r4sg9nm65qfpgcw0uy',
+    path: "m/84'/0'/0'/0/1",
+  },
+  {
+    txid: 'adb979b44c86393236e307c45f9578d9bd064134a2779b4286c158c51ad4ab05',
+    vout: 0,
+    value: '31961',
+    height: 705718,
+    confirmations: 2,
+    address: 'bc1qpszctuml70ulzf7f0zy5r4sg9nm65qfpgcw0uy',
+    path: "m/84'/0'/0'/0/1",
+  },
+]
+
 const utxoSelectInputStandard = {
+  utxos: baseUtxos,
+  to: 'bc1qppzsgs9pt63cx9x994wf4e3qrpta0nm6htk9v4',
+  satoshiPerByte: '1',
+  value: '400',
+}
+
+const utxoSelectInputAdditionalUtxo = {
   utxos: [
+    ...baseUtxos,
     {
-      txid: 'ef935d850e7d596f98c6e24d5f25faa770f6e6d8e5eab94dea3e2154c3643986',
-      vout: 0,
-      value: '1598',
-      height: 705718,
-      confirmations: 2,
-      address: 'bc1qpszctuml70ulzf7f0zy5r4sg9nm65qfpgcw0uy',
-      path: "m/84'/0'/0'/0/1",
-    },
-    {
-      txid: 'adb979b44c86393236e307c45f9578d9bd064134a2779b4286c158c51ad4ab05',
+      txid: '48cae9b44c86393236e307c45f9578d9bd064134a2779b4286c158c51ad41019',
       vout: 0,
-      value: '31961',
+      value: '100000',
       height: 705718,
       confirmations: 2,
-      address: 'bc1qpszctuml70ulzf7f0zy5r4sg9nm65qfpgcw0uy',
+      address: '15uud35JNGQboswqGeGGhTttGcX413VGcw',
       path: "m/84'/0'/0'/0/1",
     },
   ],
   to: 'bc1qppzsgs9pt63cx9x994wf4e3qrpta0nm6htk9v4',
   satoshiPerByte: '1',
-  value: '400',
+  value: '35559',
```

Comment:
we want this to be smaller than the values above (`1598` and `31961`) such that we can ensure that this has more valid utxo's to select from and the `from` filter is working correctly, say `1000` should work